docs/content/modules/tensor_mechanics/introduction/VisualizingTensors.md

```diff
@@ -0,0 +1,54 @@
+#Visualization of stresses and strains and other tensor components
+
+To visualize stresses, strains, and elasticity tensor components, the material objects must be outputted to auxiliary variables using auxiliary kernels available. RankTwoAux is used to output a RankTwoTensor component, and RankFourAux is used to output a RankFourTensor component.  For example, $\sigma_{11}, \epsilon_{22}$, and $C_{1122}$ can be visualized by first declaring auxiliary variables for them in the input file:
+
+```
+    [AuxVariables]
+      [./s11_aux]
+        order = CONSTANT
+        family = MONOMIAL
+      [../]
+
+      [./e22_aux]
+        order = CONSTANT
+        family = MONOMIAL
+      [../]
+
+      [./C1122_aux]
+        order = CONSTANT
+        family = MONOMIAL
+      [../]
+    []
+```
```

Comment:
It is possible to link to an existing input file here, rather than copy paste? I know that this syntax isn't likely to change, but you never no.